lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'dish'
+
+class Menu
+  def initialize(dish_class = Dish)
+    @dish_class = dish_class
+    @dishes = {
+      Fries: 3.0,
```

Comment:
Smart to keep it simple with the dishes.

Regarding the dishes/ .dup method... was this implemented to be able to bring up the menu again? (I'm sure you have a good reason for this but it was just something--perhaps through lack of understanding--I was curious about...)